apps/arena/native/physics/src/map.rs

```diff
@@ -51,6 +51,7 @@ pub enum Category {
     Projectile,
     Obstacle,
     PowerUp,
+    Pool,
```

Comment:
What would a `Pool` be?

Reply:
A pull is a kind of entity that does not block the player movement but apply some kind of effect, e.g. the valtimer ultimate pull the player to the center and cause damage to it